lib/Secrets/interactionparameters.h

```diff
@@ -94,6 +94,8 @@ class SAILFISH_SECRETS_API InteractionParameters {
         StoreKey            = 1 << 25,
         ImportKey           = 1 << 26,
 
+        CreatePassword      = 1 << 27,
```

Comment:
Should this be "RequestPassword" rather than "CreatePassword"?

Reply:
Ah, no I understand now: this forces the repeat-prompt.  Ignore my comment above.